install/qpc-tools

```diff
@@ -99,25 +102,43 @@ EOM
     exit 0
 }
 
-install() {
+install_cli() {
 shift
 # shift removes the $1 in the positional args
 if [[ $EUID -ne 0 ]]
 then
-  echo ansible-playbook $PLAYBOOK -vv -K $*
-  ansible-playbook $PLAYBOOK -vv -K $*
+  echo ansible-playbook cli/$CLI_PLAYBOOK -vv -K $*
+  ansible-playbook cli/$CLI_PLAYBOOK -vv -K $*
 else
-  echo ansible-playbook $PLAYBOOK -vv $@
-  ansible-playbook $PLAYBOOK -vv $*
+  echo ansible-playbook cli/$CLI_PLAYBOOK -vv $*
+  ansible-playbook cli/$CLI_PLAYBOOK -vv $*
 fi
+if [ $? -eq 0 ]
+then
+  echo "CLI Install complete."
+  exit 0
+else
+  echo "CLI Install Failed."
+  exit 1
+fi
+}
 
-unset args
+install_server() {
+shift
+if [[ $EUID -ne 0 ]]
+then
+  echo ansible-playbook server/$SERVER_PLAYBOOK -vv -K $*
+  ansible-playbook server/$SERVER_PLAYBOOK -vv -K $*
+else
+  echo ansible-playbook server/$SERVER_PLAYBOOK -vv $*
+  ansible-playbook server/$SERVER_PLAYBOOK -vv $*
+fi
 if [ $? -eq 0 ]
 then
-  echo "Installation complete."
+  echo "Quipucords Server Install complete."
```

Comment:
"Quipucords server installation complete."  or "Quipucords Server installation complete." (not sure if server should be upper/lowercase for this statement and the one below)